app/errors.py

```diff
@@ -0,0 +1,21 @@
+class VaccineError(Exception):
+    """ A template error for inherited classes 'NotVaccinatedError'
+    and 'OutdatedVaccineError' """
+
+
+class NotVaccinatedError(VaccineError):
+    """Raises if visitor don't have a 'vaccine' key in 'visitor' dict"""
+    def __str__(self) -> str:
+        return "The visitor not vaccinated"

```

Comment:
The error message in the `__str__` method for `NotVaccinatedError` should be more descriptive. Consider revising it to something like: 'The visitor is not vaccinated.'